doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,75 @@ This Appendix has two goals:
 1. Provide guidance to VNF or more generally Application vendors on how to consume CNTT Reference Model and Architectures
 2. Provide usable definitions of maturity levels for VNF software architecture between Physical-to-Virtual migration and “Cloud Native”.
 
-The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform.
+The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform. So, it will primarily look at Operational impact (in steps like Install, Configure, Validate, Operate) towards full lifecycle automation supporting CI/CD.
 
 <a name="A.2"></a>
 ## A.2 Intro and Terminology
```

Comment:
@petorre is it possible to remove this section and add those terminology to the glossary file. it is good to keep all grossly in one place.

Reply:
Yes, in #1367 .